<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditarTopico</class>
 <widget class="QDialog" name="EditarTopico">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>769</width>
    <height>561</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="3" colspan="2">
    <widget class="Line" name="line_3">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QLabel" name="ProponentesAtivos">
     <property name="text">
      <string>Topicos</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string>Especificidade:</string>
     </property>
    </widget>
   </item>
   <item row="8" column="3" colspan="2">
    <widget class="QPushButton" name="pushButton_3">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Remover</string>
     </property>
    </widget>
   </item>
   <item row="9" column="4">
    <widget class="QLineEdit" name="etRemoveQuestao">
     <property name="enabled">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="17" column="3">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Query Questões:</string>
     </property>
    </widget>
   </item>
   <item row="5" column="3">
    <widget class="QLabel" name="label_13">
     <property name="text">
      <string>Query Topico:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" rowspan="15" colspan="2">
    <widget class="QTableWidget" name="topicoTable">
     <column>
      <property name="text">
       <string>Nome</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Especificidade</string>
      </property>
     </column>
    </widget>
   </item>
   <item row="11" column="4">
    <widget class="QLineEdit" name="etRemoveTopico">
     <property name="enabled">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="14" column="3" colspan="2">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>Questões Propostas</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="15" column="3" colspan="2">
    <widget class="QTableWidget" name="questoesTable">
     <column>
      <property name="text">
       <string>Id</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Nome</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Nivel</string>
      </property>
     </column>
    </widget>
   </item>
   <item row="0" column="4">
    <widget class="QLineEdit" name="etNome"/>
   </item>
   <item row="11" column="3">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Query Topico:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2" rowspan="18">
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
   <item row="17" column="4">
    <widget class="QLineEdit" name="etQuestoes"/>
   </item>
   <item row="7" column="4">
    <widget class="QLineEdit" name="etUpdateTopico"/>
   </item>
   <item row="5" column="4">
    <widget class="QLineEdit" name="etAdicionarTopico"/>
   </item>
   <item row="9" column="3">
    <widget class="QLabel" name="label_11">
     <property name="text">
      <string>Query Questao:</string>
     </property>
    </widget>
   </item>
   <item row="17" column="1">
    <widget class="QLineEdit" name="etTopicos"/>
   </item>
   <item row="13" column="3" colspan="2">
    <widget class="Line" name="line_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="6" column="3" colspan="2">
    <widget class="QPushButton" name="pushButton">
     <property name="text">
      <string>Atualizar</string>
     </property>
    </widget>
   </item>
   <item row="4" column="3" colspan="2">
    <widget class="QPushButton" name="pushButton_2">
     <property name="text">
      <string>Adicionar</string>
     </property>
    </widget>
   </item>
   <item row="1" column="4">
    <widget class="QLineEdit" name="etEspecificidade"/>
   </item>
   <item row="0" column="3">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Nome:</string>
     </property>
    </widget>
   </item>
   <item row="10" column="4">
    <widget class="QLineEdit" name="etRemoveContempla">
     <property name="enabled">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="17" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Query Topicos:</string>
     </property>
    </widget>
   </item>
   <item row="7" column="3">
    <widget class="QLabel" name="label_9">
     <property name="text">
      <string>Query Topico:</string>
     </property>
    </widget>
   </item>
   <item row="10" column="3">
    <widget class="QLabel" name="label_12">
     <property name="text">
      <string>Query Contempla:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="4">
    <widget class="QLineEdit" name="etBuscar"/>
   </item>
   <item row="3" column="3">
    <widget class="QPushButton" name="pushButton_4">
     <property name="text">
      <string>Buscar</string>
     </property>
    </widget>
   </item>
   <item row="16" column="3">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Número de Questões:</string>
     </property>
    </widget>
   </item>
   <item row="16" column="4">
    <widget class="QLineEdit" name="etTotalQuestoes"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>EditarTopico</receiver>
   <slot>adicionar()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>653</x>
     <y>119</y>
    </hint>
    <hint type="destinationlabel">
     <x>831</x>
     <y>112</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>EditarTopico</receiver>
   <slot>atualizar()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>577</x>
     <y>183</y>
    </hint>
    <hint type="destinationlabel">
     <x>827</x>
     <y>173</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_3</sender>
   <signal>clicked()</signal>
   <receiver>EditarTopico</receiver>
   <slot>remover()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>576</x>
     <y>243</y>
    </hint>
    <hint type="destinationlabel">
     <x>795</x>
     <y>220</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_4</sender>
   <signal>clicked()</signal>
   <receiver>EditarTopico</receiver>
   <slot>buscar()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>499</x>
     <y>84</y>
    </hint>
    <hint type="destinationlabel">
     <x>815</x>
     <y>74</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>atualizar()</slot>
  <slot>adicionar()</slot>
  <slot>buscar()</slot>
  <slot>remover()</slot>
 </slots>
</ui>
